ee/src/main/java/org/jboss/as/ee/subsystem/ManagedThreadFactoryAdd.java

```diff
@@ -52,15 +50,16 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final int priority = ManagedThreadFactoryResourceDefinition.PRIORITY_AD.resolveModelAttribute(context, model).asInt();
 
         final ManagedThreadFactoryService service = new ManagedThreadFactoryService(name, jndiName, priority);
-        final ServiceBuilder<ManagedThreadFactoryImpl> serviceBuilder = context.getServiceTarget().addService(ConcurrentServiceNames.getManagedThreadFactoryServiceName(name), service);
+        final CapabilityServiceBuilder serviceBuilder = context.getCapabilityServiceTarget()
```

Comment:
Unless you are willing to refactor ManagedThreadFactoryService to support MSC 1.4.x, you need to use:
```java
context.getCapabilityServiceTarget().addCapability(ManagedThreadFactoryResourceDefinition.CAPABILITY, service);
```

Reply:
done